packages/widgetbook/test/src/addons/utils/theme_wrapper.dart

```diff
@@ -75,55 +54,62 @@ Future<void> ensureCorrectThemeIsRendered<T>({
   expect(coloredBox.color, equals(colorBlue));
 }
 
-Future<void> ensureMultipleThemesRenderedSideBySide<T>({
+Future<void> renderAddonSideBySide<T>({
```

Comment:
Why is this method generic?

Reply:
In addition, this method is quite hard to understand since it's doing so many things. It's doing: pumping, getting the build context, finding a widget and comparing. This method should make testing the side by side functionality simpler.